app/src/main/java/com/metrics_board/app/exeption/GlobalExceptionHandler.java

```diff
@@ -55,8 +55,13 @@ public ResponseEntity<ApiResponse<Void>> HttpMessageNotReadableException(HttpMes
         return badRequest("Required request body is missing");
     }
 
+    @ExceptionHandler(MissedDataToUpdateException.class)
+    public ResponseEntity<ApiResponse<Void>> MissedDataToUpdateException(MissedDataToUpdateException e) {

```

Comment:
the method name is still not correct https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html